### PR TITLE
Improve server browser backend; fix error

### PR DESCRIPTION
**Pull request by F1F7Y**
`AddMouseMovementCaptureHandler` Allows other scripts the use of `CMouseMovementCapturePanel`

### CODE REVIEW

**Comment by F1F7Y** [comment]
Fixed crash when `TAB` is pressed while in console
Replaced hardcoded filter check

**Comment by F1F7Y** [comment]
- Improves sort backend
- Fixes crash when `TAB` is pressed while console is open
- Improves MouseDelta handler logic

**Comment by F1F7Y** [comment]
Increased doubleclick by 0.1, should allow slower double clicks

File: Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut [hunk 1]
```diff
@@ -1021,32 +1023,31 @@ int function ServerSortLogic ( serverStruct a, serverStruct b)
 	var aTemp
 	var bTemp
 
-	var type = GetFocus()
 	bool direction
 
 	// We can hard code this cause adding entire columns isn't as easy
-	switch ( Hud_GetHudName( type ) ) {
-		case "BtnServerNameTab":
+	switch ( filterDirection.sortingBy ) {
```
Comment (abarichello) [inline Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut:1029]: how about also changing this to enums?

**Comment by F1F7Y** [comment]
## Final(hopefully) summary
- Improves sort backend
- Fixes crash when `TAB` is pressed while console is open
- Improves MouseDelta handler logic
- Increases doubleclick from 0.2 to 0.3
- Fixes issue #91

**Pull request merged**